DOCS/man/options.rst

```diff
@@ -6267,6 +6267,37 @@ them.
 
     macOS and cocoa-cb only.
 
+``--cocoa-cb-output-csp=<csp>``
+    This sets the color space of the layer to activate the macOS color
+    transformation. Depending on the color space used the system's EDR (HDR)
+    support will be activated. To get correct results, this needs to be set to
+    the color primaries/transfer characteristics of the VO target. It is recommended
+    to use this switch together with ``--target-trc``.
+
+    ``<csp>`` can be one of the following:
+
+    :auto:               Sets the color space to the icc profile of the
+                         screen (default).
+    :display_p3:         DCI P3 primaries, a D65 white point and the sRGB
+                         transfer function.
+    :display_p3_hlg:     DCI P3 primaries, a D65 white point and the Hybrid
+                         Log-Gamma (HLG) transfer function.
+    :display_p3_pq:      DCI P3 primaries, a D65 white point and the Perceptual
+                         Quantizer (PQ) transfer function.
+    :linear_display_p3:  DCI P3 primaries, a D65 white point and linear transfer function.
+    :dcip3:              DCI P3 color space.
+    :itur_2020:          ITU BT.2020 color space.
+    :linear_itur_2020:   ITU BT.2020 color space and linear transfer function.
+    :itur_2100_hlg:      ITU BT.2100 and the Hybrid Log-Gamma (HLG) transfer function.
+    :itur_2100_pq:       ITU BT.2100 and the Perceptual Quantizer (PQ) transfer function.
+    :itur_709:           ITU BT.709 color space.
+    :srgb:               sRGB colorimetry and non-linear transfer function.
+    :linear_srgb:        Same as sRGB but linear transfer function.
+    :generic_rgb_linear: RGB and linear transfer function.
+    :adobe_rgb1998:      Adobe RGB (1998) color space.
```

Comment:
I would unify the naming with other options like `target-prim`.

`display_p3` -> `display-p3`
`itur_709` -> `bt.709`

and so on.

Also I wonder does it really need to be set manually by user? Can't we infer those values based on `target-trc`, `target-prim` and `target-colorspace-hint` somehow?

Reply:
renamed for consistency with other options:
```
display_p3 > display-p3
display_p3_hlg > display-p3-hlg
display_p3_pq > display-p3-pq
linear_display_p3 > display-p3-linear
dcip3 > dci-p3
itur_2020 > bt.2020
linear_itur_2020 > bt.2020-linear
itur_2100_hlg > bt.2100-hlg
itur_2100_pq > bt.2100-pq
itur_709 > bt.709
srgb > srgb
linear_srgb > srgb-linear
generic_rgb_linear > rgb-linear
adobe_rgb1998 > adobe
```